internal/golangcilint/golangci_lint.go

```diff
@@ -18,190 +17,197 @@ import (
-			- github.com/onsi/ginkgo/v2
-			- github.com/onsi/gomega
-	usestdlibvars:
-		constant-kind: true
-		crypto-hash: true
-		default-rpc-path: true
-		http-method: true
-		http-status-code: true
-		sql-isolation-level: true
-		time-layout: true
-		time-month: true
-		time-weekday: true
-		tls-signature-scheme: true
-	usetesting:
-		os-temp-dir: true
-	whitespace:
-		# Enforce newlines (or comments) after multi-line function signatures.
-		multi-func: true
+  # '0' disables the following options
+  max-issues-per-linter: 0
+  max-same-issues: 0
 
 linters:
-	# We use 'disable-all' and enable linters explicitly so that a newer version
-	# does not introduce new linters unexpectedly.
-	disable-all: true
-	enable:
-		- bodyclose
-		- containedctx
-		- copyloopvar
-		- dupword
-		- durationcheck
-		- errcheck
-		- errname
-		- errorlint
-  {{- if lt .GoMinorVersion 22 }}
-		- exportloopref
-  {{- end }}
-		- exptostd
-		- forbidigo
-		- ginkgolinter
-		- gocheckcompilerdirectives
-		- goconst
-		- gocritic
-		- gofmt
-		- goimports
-		- gomoddirectives
-		- gosec
-		- gosimple
-		- govet
-		- ineffassign
-		- intrange
-		- misspell
-		- nilerr
-		- noctx
-		- nolintlint
-		- nosprintfhostport
-		- perfsprint
-		- predeclared
-		- rowserrcheck
-		- sqlclosecheck
-		- staticcheck
-		- stylecheck
-		- typecheck
-		- unconvert
-		- unparam
-		- unused
-		- usestdlibvars
-		- usetesting
-		- whitespace
+  # Disable all pre-enabled linters and enable them explicitly so that a newer version does not introduce new linters unexpectedly
+  default: none
+  enable:
+    - bodyclose
+    - containedctx
+    - copyloopvar
+    - dupword
+    - durationcheck
+    - errcheck
+    - errname
+    - errorlint
+    - exptostd
+    - forbidigo
+    - ginkgolinter
+    - gocheckcompilerdirectives
+    - goconst
+    - gocritic
+    - gomoddirectives
+    - gosec
+    - govet
+    - ineffassign
+    - intrange
+    - misspell
+    - nilerr
+    - noctx
+    - nolintlint
+    - nosprintfhostport
+    - perfsprint
+    - predeclared
+    - rowserrcheck
+    - sqlclosecheck
+    - staticcheck
+    - unconvert
+    - unparam
+    - unused
+    - usestdlibvars
+    - usetesting
+    - whitespace
+  settings:
+    errcheck:
+      check-type-assertions: false
+      # Report about assignment of errors to blank identifier.
+      check-blank: true
+      # Do not report about not checking of errors in type assertions.
+      # This is not as dangerous as skipping error values because an unchecked type assertion just immediately panics.
+      # We disable this because it makes a ton of useless noise esp. in test code.
+      {{- if .ErrcheckExcludes }}
+      exclude-functions:
+        {{- range .ErrcheckExcludes }}
+        - {{ . }}
+        {{- end }}
+      {{- end }}
+    forbidigo:
+      analyze-types: true # required for pkg:
+      forbid:
+        - pattern: ^ioutil\..*$
+        - pattern: ^http\.DefaultServeMux$
+        - pattern: ^http\.Handle(?:Func)?$
```

Comment:
We should not drop these documentation strings. The rules below set `msg:`, can we do the same here?

Reply:
The migration command dropped all comments and I needed to put them back manually. I probably missed those.